scylla/src/transport/topology.rs

```diff
@@ -549,6 +557,37 @@ fn map_string_to_cql_type(type_: &str) -> Result<CqlType, InvalidCqlType> {
     })
 }
 
+async fn query_table_partitioner(
```

Comment:
As with `query_columns`, please fetch all the contents of `scylla_tables` into a hashmap in one go.